buildSrc/src/main/kotlin/com.hedera.hashgraph.hedera-conventions.gradle.kts

```diff
@@ -45,20 +48,32 @@ repositories {
     maven {
         url = uri("https://oss.sonatype.org/content/repositories/snapshots")
     }
-    maven {
-        url = uri("https://us-maven.pkg.dev/swirlds-registry/maven-prerelease-channel")
+	maven {   
+		url = uri("https://us-maven.pkg.dev/swirlds-registry/maven-prerelease-channel")
```

Comment:
Is the whitespace here off?

Reply:
This PR adds spotless enforcement, which would make this deterministic - https://github.com/hashgraph/hedera-services/pull/2034

Reply:
Actually on this one the whitespace is correct but the PR shows it as being off.